paasta_tools/kubernetes/bin/paasta_secrets_sync.py

```diff
@@ -180,7 +189,12 @@ def main() -> None:
         vault_token_file=args.vault_token_file,
         overwrite_namespace=args.namespace,
         secret_type=args.secret_type,
-    ) else sys.exit(1)
+    )
+    timer.stop(tmp_dimensions={"result": result})
```

Comment:
question: what will prometheus do with boolean dimensions? will it convert to 0/1? I ask because we seem to have a mix of result types: some are the integer exit code and others are boolean success states